code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -553,54 +553,45 @@ obj/item/attachable/attack_hand(var/mob/user as mob)
 	name = "\improper M37 wooden stock"
 	desc = "A non-standard heavy wooden stock for the M37 Shotgun. Less quick and more cumbersome than the standard issue stakeout, but reduces recoil and improves accuracy. Allegedly makes a pretty good club in a fight too.."
 	slot = "stock"
-	wield_delay_mod = WIELD_DELAY_NORMAL
+	wield_delay_mod = WIELD_DELAY_VERY_FAST //Shotguns don't benefit from the stock nearly as much as rifles, thus the wield delay penalty is reduced.
 	icon_state = "stock"
 
 /obj/item/attachable/stock/shotgun/New()
 	..()
-	accuracy_mod = config.min_hit_accuracy_mult
+	accuracy_mod = config.low_hit_accuracy_mult
```

Comment:
Okay, here's the thing.

#1: Recoil is only really an issue with the shotgun when it's one-handed.

#2: You must wield a weapon to benefit from the stock.

#3: Scatter reduction only really matters for flechettes.

This leaves accuracy as the only real benefit for shotguns (besides a slight melee damage increment).

Reply:
Perhaps we could add some recoil to standard unmodded shotgun, wouldn't be too unrealistic.
Scatter matters for slugs as well.
The point remains.

Reply:
No it doesn't, and no, there is no reason to nerf the currently balanced shotgun because we want to arbitrarily improve some specific dimension of a particular stock.

And slug scatter being problematic is literally some blue moon shit.

Reply:
We seem to disagree. You can request the review of a different maintainer or promote a larger balance debate with the playerbase to fix this.

Reply:
Whoa! No shit.

Reply:
Look, let's playtest this, and see how it works out.

It's not going to break the game, and it's certainly better than the current status quo of stocks being pure upside when used one handed.